ml-app-deployer/src/main/java/com/marklogic/appdeployer/command/forests/DeployForestsCommand.java

```diff
@@ -122,11 +123,11 @@ protected void createForestsViaForestEndpoint(CommandContext context, List<Fores
 	 * these commands to construct a list of many forests that can be created via CMA in one request.
 	 *
 	 * @param context
-	 * @param includeReplicas This command currently doesn't make use of this feature; it's here so that other clients
-	 *                        can get a preview of the forests to be created, including replicas.
+	 * @param previewingForestCreation When previewing forest creation, replicas will be built. If not previewing
```

Comment:
I don't completely understand this param.
If I set it to true, the replicas are built? That would seem odd.